src/xdp-context.c

```diff
@@ -71,6 +71,7 @@ struct _XdpContext
   XdpPortalConfig *portal_config;
```

Comment:
Ditto with the commit message here too. `context: Store a shared proxy to the Access impl` or something like that

Reply:
You know the PR is getting too big when you don't even notice your own placeholder commit messages...